controls/V-92977.rb

```diff
@@ -99,7 +99,7 @@
           date_difference = expiration_date.mjd - password_date.mjd
           describe "Account expiration set for #{user_name}" do
             subject { date_difference }
-            it { should cmp <= input('emergency_account_period')}
+            it { should cmp <= input('emergency_account_period')/24}
```

Comment:
Should we wrap this in a floor - [geeksforgeeks.org/ruby-integer-floor-function-with-example](https://www.geeksforgeeks.org/ruby-integer-floor-function-with-example/)